Stream_support/include/CGAL/IO/io.h

```diff
@@ -55,7 +55,7 @@ IOStreams.  The basic task of such an operator is to produce a
 representation of an object that can be written as a sequence of
 characters on devices as a console, a file, or a pipe. The enum `Mode` distinguish between three different printing formats.
 
-In `ASCII` mode, numbers
+In \ascii mode, numbers
 e.g. the coordinates of a point or
```

Comment:
I do like the usage of the as the ALIASES for ASCII though as it is used just like the text ASCII it will appear here a bit different from the ways `PRETTY` and `BINARY` are presented here. I think one should distinguish between the usage of the word ASCII between the format and the parameter (as it is done here as mode).
These differences can be very tricky.